blueprints/app/files/package.json

```diff
@@ -64,7 +64,7 @@
     "webpack": "^5.74.0"
   },
   "engines": {
-    "node": "14.* || >= 16"
+    "node": "14.* || 16.* || >= 18"
```

Comment:
Note: this explicitly drops support for Node 17; we should make sure to call this out as a breaking change in the release / changelog / etc.